doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -492,18 +507,10 @@ Response:
 
 ## Entity by ID [/v2/entities/{entityId}{?attrs,options}]
 
-### Retrieve entity [GET /v2/entities/{entityId}{?attrs}]
++ Parameters
```

Comment:
It seems that the entityId is not shown in the "righ panel" operations under this ## point. Maybe parameter definition has to be done always at ### level?